GIT_AND_GITHUB.md

```diff
@@ -47,10 +47,18 @@ Use `--no-ff` when you merge to `master`.
 It's okay to just push to `master` if it's a quick fix and you really need
 it out the door. Otherwise, make it a pull request.
 
+### Pull requests
+
 Pull request descriptions should be concise and well written. The merger should
 be able to copy this description straight into the release notes instead of
 figuring out what changed or was fixed.
 
+Besides a clear description more information could be helpful, for example:
+
+- If is static UI, a screenshot would do. 
+- If is an interaction a gif would help a lot. A good tool to make gifs is [Licecap](http://www.cockos.com/licecap/).
+- If is a bug, steps to reproduce are useful to understand the context.
+
 #### Who merges the pull request?
 
```

Comment:
I am used to merge my pull requests myself. Have the reviewer review it, when its fine you comment on the PR with a  👍 , then the pull requester can merge it themselves. I like this a bit better because you stay in control of your code status. Otherwise you tend to get conversations on slack like: did you merge yet? etc.